notebooks/GPU.ipynb

```diff
@@ -1,14 +1,20 @@
 {
   "cells": [
     {
-      "cell_type": "markdown",
+      "cell_type": "code",
+      "source": [
+        "# Taken from KoboldAI's Colab.\n",
+        "# If it ain't broke don't fix it lol\n",
+        "#@title <-- Tap this if you play on Mobile { display-mode: \"form\" }\n",
+        "%%html\n",
+        "<b>Press play on the music player to keep the tab alive, then start KoboldAI below (Uses only 13MB of data)</b><br/>\n",
+        "<audio src=\"https://henk.tech/colabkobold/silence.m4a\" controls>"
+      ],
       "metadata": {
-        "id": "view-in-github",
-        "colab_type": "text"
+        "id": "k9hM83vkDfZ4"
       },
-      "source": [
-        "<a href=\"https://colab.research.google.com/github/PygmalionAI/gradio-ui/blob/master/notebooks/GPU.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
-      ]
```

Comment:
Any reason to remove the "Open in Colab" link? It's useful to people who see the notebook in GitHub first.

Reply:
Whoops. Must've not seen that. Not meant to be removed.